src/components/home/MapSection.js

```diff
@@ -6,16 +6,20 @@ import { stylesBase } from '../styledComponents';
 import map from '../../assets/homePage/map.png';
 
 const { muiGridBlockContainer } = stylesBase;
-const useStyles = makeStyles({
+const useStyles = makeStyles(theme => ({
   muiGridBlockContainer,
-  title2: { margin: '100px 0px -250px' },
+  title2: {
+    margin: '100px 0px -250px',
+    [theme.breakpoints.down('sm')]: { margin: '50px 0px -120px' },
+    [theme.breakpoints.down('xs')]: { margin: '0px 0px -50px' },
+  },
   image: {
     position: 'relative',
-    width: 'calc(100vw - 8px)',
+    width: '100%',
     objectFit: 'cover',
-    zIndex: -1,
+    zIndex: '-1',
```

Comment:
This is odd to me, `-1` as a number doesn't work here?

Reply:
oh I put it back as a number.

I thought that all fields should be a string

Reply:
Nope, you can use numbers for numeric values in React/Material UI style definitions - they get interpreted as pixels (if you want something like a percent, you need a string - `'100%'` or `'0.6em`)